aws/utility.sh

```diff
@@ -708,7 +708,16 @@ function update_cognito_userpool() {
   local userpoolid="$1"; shift
   local configfile="$1"; shift
 
-  aws --region ${region} cognito-idp update-user-pool --user-pool-id "${userpoolid}" --cli-input-json "file://${configfile}"
+  aws --region "${region}" cognito-idp update-user-pool --user-pool-id "${userpoolid}" --cli-input-json "file://${configfile}"
+}
+
+function update_userpool_client() {
```

Comment:
Time to refactor I think - utility.sh is supposed to be just that.

Pick this up when we re-organise for deploymentment framework/cloud provider